frontend/src/views/DayPicasso.vue

```diff
@@ -10,7 +10,7 @@
 </template>
 
 <script>
-import Picasso from '@/components/Picasso'
+import Picasso from '../components/Picasso'
```

Comment:
Ich würde generell eher mit `@` als mit `..` arbeiten. Dann bist du unabhängig davon, wo sich `DayPicasso` genau befindet --> weniger refactoring beim Verschieben von ganzen Vue-Komponenten

Reply:
Okay, bei mir hat PhpStorm das @ nicht aufgelöst. Konnte das Problem aber jetzt mit https://stackoverflow.com/questions/51588009/vue-js-configuring-webstorm-to-set-in-path-files-to-the-src-folder
lösen.